.github/workflows/build.yaml

```diff
@@ -0,0 +1,49 @@
+name: build
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: ghcr.io/armmbed/mbed-os-env:master-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target_board:
+        - ARCH_MAX
+        - DISCO_F746NG
+        - DISCO_F769NI
+#        - DISCO_H747I comment out because of CI error.
```

Comment:
Thanks so much for finding this issue! I feel the power of CI/GHA :D 
I created the issue about it as the record. I will try to tackle it.
https://github.com/mROS-base/mros2-mbed/issues/16

So what I want you to do is remove this line now m(_ _)m

Reply:
I checked the #16 issue.
So, I removed this board from the build matrix.